### PR TITLE
Sending prepare ride mails

### PR DESCRIPTION
**Pull request by peteruithoven**
Automatically sending a mail to drivers to prepare for their ride when the earlier order round is opened for orders. This should for example remind them to pick up the keys and arrange for a car.

Second part of: #20

### CODE REVIEW

File: webapp/ordering/cron.py [hunk 1]
```diff
@@ -170,3 +170,21 @@ def do(self):
...
+    code = 'ordering.send_prepare_ride_mails'
+
+    def do(self):
+        print("SendPrepareRideMails")
+        order_round = get_current_order_round()
+        print("Order round: %s" % order_round)
+        if order_round.is_open:
+            next_order_round = get_next_order_round()
+            print("Next order round: %s" % next_order_round)
+            if next_order_round.prepare_ride_mails_sent is False:
```
Comment (rikva) [inline webapp/ordering/cron.py]: I think this will throw an exception if there's no next order round.

Comment (peteruithoven) [reply webapp/ordering/cron.py]: True... But the same will happen when there's no current order round? I've added a check though.

Comment (rikva) [reply webapp/ordering/cron.py]: I think `get_current_order_round` is designed to always return an order round, with one exception: if there are no order rounds.

Comment (peteruithoven) [reply webapp/ordering/cron.py]: Ah alright and that's only an issue when you start a completely clean version, which is fine. 

I've added a check. Could you review again?

**Comment by rikva** [comment]
LGTM.
We should add the template first. What should it say?

**Comment by peteruithoven** [comment]
Good point. 

Title: `Automatic transport prepare reminder`
Subject: `VOKO Utrecht - Transport voorbereiding herinnering`
Body:

    Beste {{ user.first_name }},

    De bestelronde is zojuist gesloten en je staat ingeroosterd om te helpen met transport voor de volgende ronde. 
    Het is de bedoeling dat de bijrijder de sleutel ophaalt op de ophaal avond en dat de rijder ervoor zorgt dat hij de beschikking krijgt over een auto.
    Alle informatie over de rit: {{ base_url }}{% url 'ride' ride.slug %}
    Meer informatie over hoe transport werkt: https://vokoutrecht.nl/transport/

    Bedankt voor het helpen met transport!

    Groene groetjes,
    VOKO Utrecht


While writing this I noticed I wrote "ingeroosterd" wrong in the other transport reminder mail (see: https://github.com/rikva/voko/pull/33#issuecomment-479475690), could you correct that from "ingeroostered" to "ingeroosterd"?

File: webapp/ordering/cron.py [hunk 2]
```diff
@@ -170,3 +170,22 @@ def do(self):
...
+    code = 'ordering.send_prepare_ride_mails'
+
+    def do(self):
+        print("SendPrepareRideMails")
+        order_round = get_current_order_round()
+        print("Order round: %s" % order_round)
+        if order_round.is_open:
+            next_order_round = get_next_order_round()
+            print("Next order round: %s" % next_order_round)
+            if (next_order_round is not None and
```
Comment (rikva) [inline webapp/ordering/cron.py]: If the new order round is created after the current order round was closed, the mail will never be sent?

**Comment by rikva** [comment]
Done and done!

File: webapp/ordering/cron.py (hunk 2 above)
Comment (rikva) [reply webapp/ordering/cron.py]: Should it be `if order_round.is_over` on L185?

**Comment by peteruithoven** [comment]
My idea is to send this mail when they still have a change to order. For example because they have to pick up the key anyway. I realize the downside is that no mail will be send when the order round is added to late, but scheduling usually works a few months ahead.

**Comment by rikva** [comment]
That makes sense. But the email says `De bestelronde is zojuist gesloten` so I'm still a bit confused ;)

**Comment by peteruithoven** [comment]
Oh, oeps, that's what I get for coping the other mail. 

> Een nieuwe bestelronde is zojuist geopend en je staat ...

**Comment by rikva** [comment]
No worries. Updated the mail template.

**Pull request merged**